.gitignore

```diff
@@ -1,4 +1,5 @@
 target
+__pycache__
```

Comment:
what about *.pyc?

Reply:
As far as I am aware CPython 3 always writes `.pyc` files to `__pycache__`, however, I have added `*.pyc` to `.gitignore` to be sure none are commited and pushed to the repository.